### DISCUSSION TITLE
not reply to my ticket

### DISCUSSION THREAD

**Discussion by ghost** [community/community#142496] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Yes: GitHub Support does not respond even after 70 days !!

How long did it take to respond when you created a ticket on GitHub? Is this general timeframe?

i am waiting for them from long time ago , without any response

**Comment by buritbest123** [comment]
I understand how frustrating it is when GitHub Support doesn’t respond for a long time. I submitted a support ticket on **Wed, Sep 11**, but after waiting **43 days** without any response from GitHub Support, I was really frustrated.

I recently went through an issue where my **GitHub account got locked**, and I couldn’t log in after receiving a **suspicious activity notification**. Despite following the password reset instructions, I was still unable to regain access. On top of that, my account seemed to be invisible when searched by others.

Here’s how I eventually solved it: I created a **new support ticket** after realizing my original ticket wasn’t being responded to. I recommend doing this if you don’t hear back within **7 days** of submitting a ticket. Once I did this, I was able to move things forward and **regain access to my account**.

If you’re experiencing similar issues, don’t lose hope! Try creating a new ticket if your current one isn’t getting a response. I hope this helps anyone else dealing with the same problem.

**Comment by queenofcorgis** [comment]
Hi @safah222123, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket. We appreciate your understanding! I

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
